tinker/e_announcements/__init__.py

```diff
@@ -283,14 +283,14 @@ def ea_future(self):
         if 'E-Announcement Approver' not in session['groups'].split(';') and 'Administrators' not in session['groups'].split(';'):
             return abort(403)
 
-        pass_in = request.form
+        pass_in = EncodingDict(request.form)
         date_id = pass_in.get('dateId', 'null')
         ea_display = []
 
         forms = self.base.traverse_xml(app.config['E_ANNOUNCEMENTS_XML_URL'], 'system-block', True)
         forms.sort(key=lambda item: ['created_on'], reverse=True)
 
-        def get_title_and_message(self, form):
+        def get_title_and_message(form):
```

Comment:
I found this `self,` arg superfluous, removed it, and unit/integration tests still pass.